homeassistant/components/avri/sensor.py

```diff
@@ -20,6 +20,7 @@
 ICON = "mdi:trash-can-outline"
 SCAN_INTERVAL = timedelta(hours=4)
 DEFAULT_COUNTRY_CODE = "NL"
+UNIQUE_ID_DELIMITER = "_"
```

Comment:
A constant is not really necessary for a unique id, see the example in the [docs](https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements).

Reply:
:+1: